### PR TITLE
Feat/list component

### PR DESCRIPTION
**Pull request by m-matheus**

### CODE REVIEW

File: apps/agile-project-fe/src/app/components/list/list.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
```
Comment (guilhXavier) [inline apps/agile-project-fe/src/app/components/list/list.stories.tsx:1]: Why do we need to disable eslint here?

File: apps/agile-project-fe/src/app/components/list/list.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      defaultValue: [
+        { name: 'ChipIn 1', amount: 100, user: { name: 'User 1' } },
+        { name: 'ChipIn 2', amount: 200, user: { name: 'User 2' } },
+      ],
+    },
+  },
+};
+
+export const Primary = ({
+  items = [
```
Comment (guilhXavier) [inline apps/agile-project-fe/src/app/components/list/list.stories.tsx:22]: Please put this default list into it's own `const` so it'll be easier to read

File: apps/agile-project-fe/src/app/components/list/list.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import { StyledList } from './list.styled';
+
+export interface ListProps {
+  items: Array<ChipIn>;
+}
+
+export const List: React.FC<ListProps> = ({ items }) => (
+  <StyledList>
+    <ul>
+      {Array.isArray(items) ? (
```
Comment (guilhXavier) [inline apps/agile-project-fe/src/app/components/list/list.tsx:12]: This check is redundant since we already defined items as an array

**Review by guilhXavier — CHANGES_REQUESTED** [review]

File: apps/agile-project-fe/src/app/components/list/list.stories.tsx (hunk 1 above)
Comment (m-matheus) [reply apps/agile-project-fe/src/app/components/list/list.stories.tsx:1]: We don't need it, the file was generated with this and I forgot to remove it. Sorry about that.

**Review by guilhXavier — APPROVED** [review]
Nice work!

**Pull request merged**